spec/GraphQLOverHTTP.md

```diff
@@ -158,7 +158,7 @@ With the following query variables:
 
 ```graphql example
```

Comment:
The language could be improved too:

```suggestion
```json example
```

Reply:
Agree ! Or maybe "variables as json input example" ?